FirstApp/Favourites/ProfileRepo.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  Trying.swift
+//  FirstApp
+//
+//  Created by Akua Afrane-Okese on 2020/04/27.
+//  Copyright © 2020 Akua Afrane-Okese. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc public class ProfileRepo: NSObject {
+    @objc(save:) public func save(name: NSData) {
+        guard let appDelegate =
+          UIApplication.shared.delegate as? AppDelegate else {
+          return
+        }
+        let managedContext =
+          appDelegate.persistentContainer.viewContext
+        let entity =
+          NSEntityDescription.entity(forEntityName: "Camera",
+                                     in: managedContext)!
+        let photo = NSManagedObject(entity: entity,
+                                     insertInto: managedContext)
+        photo.setValue(name, forKeyPath: "photo")
+        do {
+          try managedContext.save()
+        } catch let error as NSError {
+          print("Could not save. \(error), \(error.userInfo)")
```

Comment:
Rather display the error with an alert, instead of printing

Reply:
Interesting idea. Thanks, I'll look into it when I get back to this task again.

Reply:
Cool, if you don't wanna display and it needs to have code in the braces, use return